src/ergw_aaa_radius.erl

```diff
@@ -359,8 +363,12 @@ vendor_ituma('WLAN-Station-RSSI ', #{'WLAN-Station-RSSI' := Value}, Attrs) ->
     [{?'IM_UE_Sta_RSSI', Value}|Attrs];
 vendor_ituma('WLAN-AP-RSSI', #{'WLAN-AP-RSSI' := Value}, Attrs) ->
     [{?'IM_UE_Wwan_RSSI', Value}|Attrs];
+vendor_ituma('LI-Location', #{'IM_LI_Location' := LocationStr}, Attrs) ->
```

Comment:
Perhaps `Location` is a better variable name?

Reply:
I wanted to emphasize that it's a single string and not a gps formatted string. But yes, I can do that 👍

Reply:
Perhaps add GPS to the GPS location name? And leave this as location, only.

Reply:
Hmmmm, it's already like "Latitude" and "Longitude". NP, LocationStr -> Location.

Reply:
Erlang/OTP does not use nor suggest anything like Hungarian" notation conventions or any other variable type annotation.
I don't see any reason to start with that either. Name variables after the semantic meaning of their content, not after their type. `Location` is perfectly fine, not need to a `GPS` or `Str`.

Note: RADIUS AVP have different semantics and naming coventions.

Reply:
ok